client/components/NewBankForm.js

```diff
@@ -112,19 +129,38 @@ export default class NewBankForm extends React.Component {
                 <option key={bank.id} value={bank.uuid}>{bank.name}</option>
             );
 
-            var maybeWebsites;
-            if (this.state.hasWebsites) {
-                var websitesOptions = this.state.websites.map((website) =>
-                    <option key={website.hostname} value={website.hostname}>{website.label}</option>
-                );
-                maybeWebsites = <div className="form-group">
-                    <label htmlFor="website"><T k='settings.website'>Website</T></label>
-                    <select className="form-control" id="website" ref="website">
-                        {websitesOptions}
-                    </select>
-                </div>;
+            var maybeCustomFields= [];
+            if (this.state.hasCustomFields) {
+
+                maybeCustomFields = this.state.customFields.map(function(field, index) {
+                    var customFieldFormInput;
+                    var customFieldFormInputId = "customField" + index;
+
+                    switch (field.type)
```

Comment:
please add a `default` case to this switch, that calls `alert` with an error message such as `unknown field type`